app/domain/authentication/o_auth/fetch_provider_certificates.rb

```diff
@@ -13,7 +13,7 @@ module AuthnOidc
     FetchProviderCertificates = CommandClass.new(
       dependencies: {
         logger:                 Rails.logger,
-        discover_oidc_provider: Authentication::AuthnOidc::DiscoverOIDCProvider.new
+        discover_oidc_provider: ::Authentication::OAuth::DiscoverIdentityProvider.new
```

Comment:
Would you be able to also reference a file from the same level?

Ex: `DiscoverIdentityProvider.new` instead of `::Authentication::OAuth::DiscoverIdentityProvider.new`

Reply:
Yes.

Reply:
so i think `DiscoverIdentityProvider.new` would be cleaner